lib/oauth2.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+#Complete OAuth2.0
+#
+from httplib2 import Http
+from oauth2client import file, client, tools
+import yaml
+import json
+from imaplib import IMAP4_SSL
+def authentication():
+   SCOPES = 'https://mail.google.com/'
```

Comment:
if we're hardcoding gmail (which we are elsewhere, so not a big deal) we might as well hardcode the gmail IMAP urls as well